### PR TITLE
Current speed and acceleration in status

### PR DESCRIPTION
**Pull request by zztopper**
I removed X-Y coords, moved Z ccord to the left, moved extrusion percentage and fan speed into old Y and Z positions, and added current gcode_move speed in mm\s and current max acceleration into old extrusion and fan positions.

### CODE REVIEW

**Comment by zztopper** [comment]
Commit history is dirty, but file changes are clean.

File: panels/job_status.py [hunk 1]
```diff
@@ -136,18 +136,27 @@ def initialize(self, panel_name):
         self.labels['it_box'] = it_box
 
         position = self._gtk.Image("move.svg", None, .6, .6)
-        self.labels['pos_x'] = Gtk.Label(label="X: 0")
-        self.labels['pos_x'].get_style_context().add_class("printing-info")
-        self.labels['pos_y'] = Gtk.Label(label="Y: 0")
-        self.labels['pos_y'].get_style_context().add_class("printing-info")
-        self.labels['pos_z'] = Gtk.Label(label="Z: 0")
+        self.labels['pos_z'] = Gtk.Label(label="X: 0")
```
Comment (alfrix) [inline panels/job_status.py:139]: pos_z labeled as X:

**Comment by alfrix** [comment]
> Commit history is dirty, but file changes are clean.

repo owners can squash your changes, but if you can cleanup the history if you want, just do a `rebase -i @hash` or  `rebase -i @~numberofcommits` and then `push -f`

File: panels/job_status.py (hunk 1 above)
Comment (zztopper) [reply panels/job_status.py:139]: Thanks, will fix that now

**Pull request closed without merging**